packages/flutter/lib/src/rendering/image_filter_config.dart

```diff
@@ -0,0 +1,182 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui' as ui;
+
+import 'package:flutter/foundation.dart';
+
+/// A configuration for a [ui.ImageFilter].
+///
+/// This class provides a framework-level abstraction for image filters that can
```

Comment:
Just curious have you thought about renaming existing `ImageFilter` to something else and reuse the name for this class?

Reply:
That's quite a clever duck typing. Well I can think of one way it can break: `ImageFilter` actually comes from `dart:ui` and people might write import `dart:ui` as `ui` and write `ui.ImageFilter` instead.

Reply:
That won't be a problem if this is definded in dart:ui right? though that means we need to define both web and not web version of this